pydm/utilities/stylesheet.py

```diff
@@ -0,0 +1,123 @@
+# Utility to handle importing the global stylesheet for PyDM widgets
+
+from ..PyQt.QtGui import QApplication
+import os
+from functools import partial
+
+import logging
+logger = logging.getLogger(__name__)
+from . import is_pydm_app
+
+
+# Fallback global stylesheet if there is no global stylesheet provided via env variable or command line parameter
+# TODO: Expand this global stylesheet
+GLOBAL_STYLESHEET = "QLabel {" \
```

Comment:
This should probably be a separate file that gets bundled with PyDM.  And we definitely need a real default stylesheet, blue and orange background colors will have people very confused.  Or maybe blind.

Reply:
Being blinded with science may sound cool in the '80s but it's not the goal here. I've added the code to import a default stylesheet externally.